web-test-runner.config.mjs

```diff
@@ -1,7 +1,22 @@
+import { playwrightLauncher } from '@web/test-runner-playwright';
+
 export default {
+  browsers: [
+    playwrightLauncher({
+      createBrowserContext: async ({ browser, config }) => {
+        const context = await browser.newContext();
+        // grant permissions to access the users clipboard
+        await context.grantPermissions(['clipboard-read', 'clipboard-write']);
+        return context;
+      },
+    }),
+  ],
   testFramework: {
     config: {
-      ui: 'bdd'
+      ui: 'bdd',
+      // override default of 2000 miliseconds to support testing
+      // state changes in pfe-clipboard
+      timeout: '3500',
```

Comment:
Is it possible to only change the timeout for testing within pfe-clipboard instead? If this timeout is needed for pfe-clipboard specifically, I'd prefer to change it there instead of globally. 

Like you say it's prob not a big deal to make this change global except that tests will just take longer before they fail.

Reply:
Looks like this may work if you want to go this way 

```js
describe('a slow thing', () => {
 // test code...
}).timeout(5000);```

Reply:
Yeah good question; I would prefer that as well.  I couldn't find a way to change it on the component level but I'm sure it's possible.  Maybe a question @bennypowers would know the answer to.

Reply:
for reference
https://mochajs.org/#arrow-functions

Reply:
I think the main reason it's not an issue with Cypress is you rarely need to access Mocha's context.

Reply:
Sorry @bennypowers, you were too quick.  I didn't see your message before I sent it.  I'm good with changing them from arrow functions.